### PR TITLE
Document how to trigger phx-events from JavaScript

### PR DESCRIPTION
**Pull request by Gazler**

### CODE REVIEW

File: guides/client/js-interop.md [hunk 1]
```diff
@@ -122,6 +122,29 @@ In the case of an `"element"` page loading event, the info will contain a
...
+    )
+
+When using a client hook, `this.el` can be used to determine the element as
+outlined in the "Client hooks" documentation.
+
+It is also possible to trigger a `phx-submit` using a "submit" event:
+
+    document.getElementById("my-form").dispatchEvent(
+      new Event("submit", {bubbles: true})
+    )
```
Comment (josevalim) [inline guides/client/js-interop.md:146]:
```suggestion
```javascript
document.getElementById("my-form").dispatchEvent(
  new Event("submit", {bubbles: true})
)
```
```

Comment (josevalim) [inline guides/client/js-interop.md:146]: We probably want to wrap those in triple quotes + javascript  blocks. :)

**Review by josevalim — APPROVED** [review]

File: guides/client/js-interop.md [hunk 2]
```diff
@@ -122,6 +122,29 @@ In the case of an `"element"` page loading event, the info will contain a
 `"target"` key containing the DOM element which triggered the page loading
 state.
 
+## Triggering phx events with JavaScript
```
Comment (chrismccord) [inline guides/client/js-interop.md:125]:
```suggestion
## Triggering phx form events with JavaScript
```

File: guides/client/js-interop.md [hunk 3]
```diff
@@ -122,6 +122,29 @@ In the case of an `"element"` page loading event, the info will contain a
...
+
+Often it is desirable to trigger an event on a DOM element without explicit
```
Comment (chrismccord) [inline guides/client/js-interop.md:127]:
```suggestion
Often it is desirable to trigger an event on a form element without explicit
```

Comment (Gazler) [reply guides/client/js-interop.md]: Technically the change event is on the input and not the form element, so this might add confusion.

**Pull request merged**